### ISSUE TITLE
Laravel: An error occurs when post data like "123"

### ISSUE THREAD

**Issue by Mradxz** [itsgoingd/clockwork#492]
```
POST /api/test HTTP/1.1
Connection: close
Content-Length: 3
Host: 127.0.0.1:8000

123
```

```
{
    "message": "array_map(): Expected parameter 2 to be an array, int given",
    "exception": "ErrorException",
    "file": "/Users/mac/code/WowFaker/vendor/itsgoingd/clockwork/Clockwork/Helpers/Serializer.php",
    "line": 102,
    "trace": [
        {
            "function": "handleError",
            "class": "Illuminate\\Foundation\\Bootstrap\\HandleExceptions",
            "type": "->"
        },
        {
            "file": "/Users/mac/code/WowFaker/vendor/itsgoingd/clockwork/Clockwork/Helpers/Serializer.php",
            "line": 102,
            "function": "array_map"
        },
        {
            "file": "/Users/mac/code/WowFaker/vendor/itsgoingd/clockwork/Clockwork/DataSource/PhpDataSource.php",
            "line": 55,
            "function": "normalizeEach",
            "class": "Clockwork\\Helpers\\Serializer",
            "type": "->"
        },
        {
            "file": "/Users/mac/code/WowFaker/vendor/itsgoingd/clockwork/Clockwork/DataSource/PhpDataSource.php",
            "line": 20,
            "function": "getRequestData",
            "class": "Clockwork\\DataSource\\PhpDataSource",
            "type": "->"
        },
        {
            "file": "/Users/mac/code/WowFaker/vendor/itsgoingd/clockwork/Clockwork/Clockwork.php",
            "line": 59,
            "function": "resolve",
            "class": "Clockwork\\DataSource\\PhpDataSource",
            "type": "->"
        },
        {
            "file": "/Users/mac/code/WowFaker/vendor/itsgoingd/clockwork/Clockwork/Support/Laravel/ClockworkSupport.php",
            "line": 366,
            "function": "resolveRequest",
            "class": "Clockwork\\Clockwork",
            "type": "->"
        },
```

**Comment by itsgoingd** [comment]
Fixed in master, thanks!

**Comment by itsgoingd** [comment]
Released as part of 5.0.8.
